tools/create-vuepress/src/flow/createPackageJson.ts

```diff
@@ -0,0 +1,108 @@
+import { writeFileSync } from 'node:fs'
+import { join } from 'node:path'
+import inquirer from 'inquirer'
+import type { CreateLocaleOptions } from '../i18n/index.js'
+import type { PackageManager } from '../utils/index.js'
+
+const PACKAGE_NAME_REG =
+  /^(?:@[a-z0-9-*~][a-z0-9-*._~]*\/)?[a-z0-9-~][a-z0-9-._~]*$/u
+
+const VERSION_REG = /^[0-9]+\.[0-9]+\.(?:[0-9]+|[0-9]+-[a-z]+\.[0-9])$/u
+
+interface CreatePackageJsonOptions {
+  targetDir: string
+  packageManager: PackageManager
+  locale: CreateLocaleOptions
+  preset: 'blog' | 'docs'
+  bundler: 'vite' | 'webpack'
+}
+
+interface PackageJsonAnswer {
+  name: string
+  version: string
+  description: string
+  license: string
+}
+
+/**
+ * generate package.json
+ */
+export const createPackageJson = async ({
+  targetDir,
+  packageManager,
+  locale,
+  preset,
+  bundler,
+}: CreatePackageJsonOptions): Promise<void> => {
+  const packageJsonPath = join(targetDir, 'package.json')
+  // TODO: Update it
```

Comment:
You can make use of `npm view vuepress@next version` command to get the latest version

Reply:
I am negative towards it, a lot of area in China can counter timeout at this stage.